stonesoup/sensor/radar/radar.py

```diff
@@ -163,6 +163,9 @@ def rotate(self, timestamp):
             A timestamp signifying when the rotation completes
         """
 
+        if self.dwell_center.timestamp == None:
```

Comment:
Changing `==` to `is` should work.
```suggestion
        if self.dwell_center.timestamp is None:
```